src/features/Instructors/AddInstructors/index.jsx

```diff
@@ -31,18 +32,34 @@ const initialState = {
     email: '',
     firstName: '',
     lastName: '',
+    hasEnrollment: false,
   },
 };
 
 const AddInstructors = ({ isOpen, onClose }) => {
+  const enableEnrollmentToggle = getConfig()?.ENABLE_ENROLLMENT_TOGGLE || false;
```

Comment:
Why don't you use the SHOW_INSTRUCTOR_FEATURES flag? these tickets are in the same epic and the idea is when all are ready move at the same time